index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Pirate Bartender</title>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" type="text/css" href="css/main.css">
+	<link rel="stylesheet" type="text/css" href="css/animate.min.css">
+	<link rel="stylesheet" type="text/css" href="css/normalize.css">
+
+</head>
+<body>
+
+<div class="container">
+
+	<div class="entrance">
+		<div class="content">
+			<h2>Welcome to the Pirate Bar!</h2>
+			<p>Quench yer mighty thirst with a beverage fit fer a salty dog!</p>
+			<button class="enter-button">Enter At Yer Own Risk!</button>
+		</div>
+	</div>
+
+	<div class="inside">
+	<div class="pirate-questions"></div>
+	<button id="btn-yes">Yes</button>
+	<button id="btn-no">No</button>
+	<button id="btn-answer">Tell me what he said!</button>
+
+	<!-- <form class="response">
+		<label>Answer the Pirate</label>
+		<input type="text" name="preferences">
+	</form> -->
+
+	</div>
+	
+
+
+
+</div>
+
+
+
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
```

Comment:
don't need a type= here, but if you decide to keep it, add it to all the script tags